adagrams/game.py

```diff
@@ -1,13 +1,100 @@
 from random import randint
 
+
 def draw_letters():
-    pass
+    #copy letter_pool from test and change it from dic to list
+    #new variable
+    letter_pool = ['A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 'B', 'B', 'C', 'C', 'D', 'D', 'D', 'D',
+ 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'F', 'F', 'G', 'G', 'G',
+ 'H', 'H', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 'J', 'K', 'L', 'L', 'L', 'L',
+ 'M', 'M', 'N', 'N', 'N', 'N', 'N', 'N', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'P',
+ 'P', 'Q', 'R', 'R', 'R', 'R', 'R', 'R', 'S', 'S', 'S', 'S', 'T', 'T', 'T', 'T', 'T',
+ 'T', 'U', 'U', 'U', 'U', 'V', 'V', 'W', 'W', 'X', 'Y', 'Y', 'Z']
+
+    drawn_letters = []
+    #using the row with only from 10 letters that in the letterpool
+    for i in range(10):
```

Comment:
Since we declare i but don't access it, prefer to use _ instead.
```suggestion
    for _ in range(10):
```

Reply:
If I did not know how to play Adagrams, I would not understand the significance of 10. If I were a dev who tried to read this, I might be wondering "Why do we only loop 10 times"?

It would be more descriptive to use a constant variable to reference 10 so that your code is more self-documenting. 

```py
NUM_TILES_ALLOWED_IN_HAND = 10
for _ range(NUM_TILES_ALLOWED_IN_HAND):
```